hack/make/debug/debug.mk

```diff
@@ -0,0 +1,40 @@
+
+## Run the operator locally
+debug/operator:
+	kubectl -n dynatrace scale --replicas=0 deployment/dynatrace-operator
+	POD_NAMESPACE=dynatrace RUN_LOCAL=true go run ./cmd operator
+
+## Run the webhook locally (requires running telepresence)
+debug/webhook:
+	env $$(cat local/telepresence.env | xargs) go run ./cmd webhook-server --certs-dir=./local/certs/
+
+## Build image with debugging information and dlv debugger (required for CSI debugging)
+debug/build:
+	DEBUG=true make images/build/push
+
+## Deploy the operator with modified csi-driver and webhook (requires image built with debug information)
+debug/deploy:
+	DEBUG=true make deploy/helm
+
+## install and start telepresence in the cluster
+debug/telepresence/install:
+	telepresence helm install --create-namespace
+	telepresence connect -n dynatrace
+	telepresence intercept dynatrace-webhook -p 8443 --env-file local/telepresence.env
+
+## stop telepresence
+debug/telepresence/stop:
```

Comment:
nitpick:
either call it `debug/telepresence/install` and `debug/telepresence/uninstall`
or `debug/telepresence/start` and `debug/telepresence/stop`